java/src/main/java/com/google/oak/crypto/DecryptionResult.java

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.google.oak.crypto;
+
+public final class DecryptionResult {
```

Comment:
Shouldn't this be used as input to the encrypt function too? (and renamed)

Reply:
`DecryptionResult` is pre-existing, it just was moved to a separate file.
Will use it as an input argument in a separate PR.